ui/lib/pki/addon/components/page/pki-configuration-edit.ts

```diff
@@ -43,28 +43,41 @@ export default class PkiConfigurationEditComponent extends Component<Args> {
   @service declare readonly version: VersionService;
 
   @tracked invalidFormAlert = '';
-  @tracked errorBanner = '';
+  @tracked errorBanner: object[] = [];
```

Comment:
Small nit but `errors` would better describe what this variable represents to me but personal preference. Pluralizing makes me think array right away. I guess with ts maybe it doesn't matter as much since you have the type annotation.

Reply:
Cool I think I typically use `errors` since a component doesn't have an `errors` property that will conflict.

Reply:
Gotcha, thanks!